llama_finetuning.py

```diff
@@ -12,21 +12,23 @@
 from torch.distributed.fsdp import (
     FullyShardedDataParallel as FSDP,
 )
+from torch.distributed.fsdp.fully_sharded_data_parallel import CPUOffload
```

Comment:
You can import CPUOffload directly from torch.distributed.fsdp together in line 12